src/components/presentation/shareButton/Share.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FacebookShareButton,
+  FacebookIcon,
+  TwitterShareButton,
+  TwitterIcon,
+  GooglePlusShareButton,
+  GooglePlusIcon,
+} from 'react-share';
+
+class Share extends React.Component {
+  state = {
+    toggleShare: false,
+  }
+
+  onShare = () => {
+    this.setState(prevState => ({
+      toggleShare: !prevState.toggleShare,
+    }));
+  }
+
+  render() {
+    const { REACT_APP_URL } = process.env;
+    const { articleSlug, articleTitle } = this.props;
+    const url = `${REACT_APP_URL}/articles/${articleSlug}`;
+    const title = articleTitle;
+    const { toggleShare } = this.state;
+    return (
+      <React.Fragment>
```

Comment:
Hi @frostyblok, I can see you're not handling for sharing via emails as well, any particular reason for that?

Reply:
No standout reason for that, I just included that to allow the users to share articles on more platforms.

Reply:
Alright then 👍